other/cross-chain/multichain-setup.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Angle Multichain setup for bridging
+---
+
+# Angle Multichain setup
+
+The first bridging solution adopted by Angle to bridge tokens was Multichain. Here is the bridge setup for agEUR and ANGLE when using Multichain. 
+
+## Bridge Solutions
+
+| Chain         | Bridge                                                                 | agEUR  | ANGLE         |
```

Comment:
I'd keep this table with for each token the list of supported tokens and potentially say whether native and direct wrapping is supported or not

Reply:
What do you mean by this?

Reply:
Like we still need an informative table that says for each chain which bridges are supported, and if people using these bridges will have to do the wrapping or not

Reply:
For some bridges you'll still have to do it. We won't promote it, but it'll be a possibility. Like Multichain